packages/adapter/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@chainsafe/nearsnap-adapter",
```

Comment:
do we need this package? Considering we can  integrate directly with wallet-selector